[CS2113-T12-1] ChChing

Rayleigh47 opened the pull request:
ChChing is a desktop app for tracking spending, and it uses a Command Line Interface (CLI) for managing finances. If you are someone who needs a simple interface to get a better hold of your finances, this app is for you!

AkmalHanis commented on docs/DeveloperGuide.md:
````diff
@@ -8,23 +8,119 @@
+The main class in our program is the ```Record``` and ```RecordList``` abstract classes, in which ```Income```, ```Expense``` will inherit from ```Record``` and ```IncomeList``` and ```ExpenseList``` will inherit from ```RecordList```. Most commands will act on instances of the ```Income```, ```Expense```,```IncomeList``` and ```ExpenseList``` classes.  
+
+![Record Class](images/Record_RecordList_UML_class.png)
+
````
For this UML diagram:

1. Abstract class for RecordList and Record is incorrect. Instead of << >> , it should be denoted as {RecordList} instead?
2. The font size of the diagram is a little bit small, making it blurry and hard to read. Perhaps this could be changed to make it easier to read.

AkmalHanis commented on docs/DeveloperGuide.md:
```diff
@@ -8,23 +8,119 @@
+<br> ![edit income sequence diagram](images/EditIncomeCommand_sequence_diagram.png)
+
+The edit expense command works in a similar way, with its sequence diagram as shown:
+<br> ![edit expense sequence diagram](images/EditExpenseCommand_sequence_diagram.png)
```
This edit expense sequence diagram may not be necessary due to its similarity to the edit income sequence diagram shown above. On the other hand, this font is clearer and more comprehensive to read as compared to the above sequence diagrams.

AkmalHanis commented on docs/DeveloperGuide.md:
````diff
@@ -8,23 +8,119 @@
+   is called, which iterates through the expenseList, ```expenses``` and prints the index as well as a completed string of
+   expenses in ```expenses```.
+
+![ListExpenseCommand](images/ListExpenseCommand_Sequence_Diagram.png)
````
For this diagram:
1. This image is named as a sequence diagram when it is actually a class diagram
2. Other than showing ListExpenseCommand class inherits from Command class, this does not add much significance at this point of the dg
3. Possible to move this class diagram up above

AkmalHanis replied:
The execute portion of the class diagram is also incorrect. It should be something like execute( ... ): void.

AkmalHanis commented on docs/DeveloperGuide.md:
````diff
@@ -8,23 +8,119 @@
+Afterwards, the ```execute()``` method will call the ```printSelector()``` method from ```Selector```.
+The ```printSelector()``` method will print all the available currencies in the selector hashmap.
+The selected currencies will be marked with a ```[X]``` and the unselected currencies will be marked with a ```[ ]```.
+
````
Formatting in the DG is a little bit inconsistent. While ListExpenseCommand and EditIncomeCommand uses numbering to sequence the explanation, the other Commands are explained in one large chunk. It may be better to standardize the way of explaining.

AkmalHanis reviewed (commented):
The DG is incomplete. Non-Functional Requirements and Instructions for manual testing are not completed.
It may be better to focus on other classes rather than just all the Command classes. Storage, UI and Parser could be explained in the DG as well.

lipkuangchng commented on docs/DeveloperGuide.md:
````diff
@@ -8,23 +8,128 @@
+### Record and RecordList  
+
+The main class in our program is the ```Record``` and ```RecordList``` abstract classes, in which ```Income```, ```Expense``` will inherit from ```Record``` and ```IncomeList``` and ```ExpenseList``` will inherit from ```RecordList```. Most commands will act on instances of the ```Income```, ```Expense```,```IncomeList``` and ```ExpenseList``` classes.  
+
````
The notation for abstract class for Record and RecordList is wrong. It should be {Abstract} rather than << Abstract >>. 

Overall, a good detailed class diagram provided

lipkuangchng commented on docs/DeveloperGuide.md:
```diff
@@ -8,23 +8,128 @@
+### Target and TargetStorage
+
+The `Target` and `TargetStorage` class allows users to set a target for their ideal balance.
+
```
Good use of protected variables as seen from the # notation.

lipkuangchng commented on docs/DeveloperGuide.md:
```diff
@@ -8,23 +8,128 @@
 {Describe the design and implementation of the product. Use UML diagrams and short code snippets where applicable.}
 
+## Implementation
+
```
You could add the system architecture diagram in the next iteration of this DG to depict the high-level design of your software and make it more detailed.

lipkuangchng commented on docs/DeveloperGuide.md:
```diff
@@ -8,23 +8,128 @@
+| v2.0    | user      | set a target for my balance             | improve my financial management                                                         |
+| v2.0    | user      | see the target i have set               | remind myself of my target                                                              |
+| v2.0    | user      | reset my income/expense lists or both   | have a fresh list                                                                       |
 ## Non-Functional Requirements
```
Please add these details in the next iteration in your DG

lipkuangchng replied:
Eg. Add sample test cases for commands

lipkuangchng commented:
Overall, a detailed DG draft. Great job! Do note to correct the minor errors, add on to manual testing as well as provide a system architecture diagram. Hope to see more improvements in the next iteration of the DG.

TJ-Hoo commented on docs/DeveloperGuide.md:
```diff
@@ -8,23 +8,128 @@
 ### Target user profile
 
-{Describe the target user profile}
+Target users are people who are keen on improving their financial accountability
```
Could this be elaborated on further? For example, what exactly is financial accountability?

WilsonLee2000 replied:
To add on the above. Perhaps a brief description of how this product is able to help people to improve their financial accountability can be briefly stated?

TJ-Hoo commented on docs/DeveloperGuide.md:
```diff
@@ -8,23 +8,128 @@
 ### Value proposition
 
-{Describe the value proposition: what problem does it solve?}
+The value proposition of ChChing is its ability to track income and expenses on a daily basis.
```
Could this be elaborated on further? How else does this application solve the problem better than other similar applications?

TJ-Hoo commented on docs/DeveloperGuide.md:
```diff
@@ -8,23 +8,128 @@
-|v1.0|new user|see usage instructions|refer to them when I forget how to use the application|
-|v2.0|user|find a to-do item by name|locate a to-do without having to go through the entire list|
-
+| Version | As a ...  | I want to ...                           | So that I can ...                                                                       |
+|---------|-----------|-----------------------------------------|-----------------------------------------------------------------------------------------|
+| v1.0    | new user  | see usage instructions                  | refer to them when I forget how to use the application                                  |
+| v1.0    | user      | add new expense to the records          | record all my expenses                                                                  |
+| v1.0    | user      | add new income to the records           | record all my incomes                                                                   |
+| v1.0    | user      | view all the records                    | refer to them when I forgot my expenses and incomes                                     |
+| v1.0    | user      | edit the records                        | modify/fix if the records is changed/wrong                                              |
+| v1.0    | user      | know current balance                    | aware how much money do I have left                                                     |
+| v2.0    | user      | edit my existing entries                | rectify or update any entries without having to enter another entry and delete an entry |
+| v2.0    | user      | find specific entries                   | refer to specific entries when necessary                                                |
+| v2.0    | foreigner | view my entries in a different currency | read my incomes and expenses in a currency I am familiar with                           |
+| v2.0    | user      | set a target for my balance             | improve my financial management                                                         |
+| v2.0    | user      | see the target i have set               | remind myself of my target                                                              |
+| v2.0    | user      | reset my income/expense lists or both   | have a fresh list                                                                       |
```
Great user stories!! Your application accurately solves the problems listed.

WilsonLee2000 replied:
I like how you implement the user stories in this format, and user stories are clear and specific.

TJ-Hoo commented on docs/DeveloperGuide.md:
````diff
@@ -8,23 +8,128 @@
+The ```printSelector()``` method will print all the available currencies in the selector hashmap.
+The selected currencies will be marked with a ```[X]``` and the unselected currencies will be marked with a ```[ ]```.
+
+![SetCurrencyCommand_sequence_diagram.png](images%2FSetCurrencyCommand_sequence_diagram.png)
````
To add on to the above, the formating of the sequence diagrams themselves are slightly inconsistent, in terms of the colour of the boxes of the instances, and the word font colour of the operations.

WilsonLee2000 commented on docs/DeveloperGuide.md:
````diff
@@ -8,23 +8,128 @@
+
+The proposed DeleteIncomeCommand mechanism is facilitated by ```System```, ```Incomes```, ```UI``` and ```ChChingException```.  
+The command receives the instruction from ```UI``` and will call the ```execute``` method.  
+If ```index <= 0```, the command will throw a new ChChingException and print ```"Zero/Negative index"```.  
````
Should the arrow for this part of the sequence diagram ChChingException("Zero/Negative index") be further improved?

WilsonLee2000 commented on docs/DeveloperGuide.md:
```diff
@@ -8,23 +8,128 @@
+* **Alternative 2:** Edit to require user to rewrite all its fields.
+    * Pros: Easier to implement.
+    * Cons: Not any easier than having the user to just delete and add new expense/income.
+
```
I really like how you implement and included this activity diagram as part of the Developer's Guide to give a clearer view of what is happening when the user executes the edit income command. The activity diagram is also accurate and neat. Well Done!

hyperbola-bear committed:
Merge pull request #12 from AY2223S2-CS2113-T12-1/master

update my repo

avielcx committed:
Merge pull request #242 from thomasjlalba/Thomas-PPP

Update Thomas's Project Portfolio Page

avielcx committed:
Update UG and DG

avielcx committed:
Merge pull request #243 from avielcx/master

Update UG and DG

avielcx committed:
Fix Storage bug

avielcx committed:
Merge pull request #244 from avielcx/master

Fix Storage bug

Rayleigh47 committed:
added PPP

thomasjlalba committed:
Update runtest's expected output to allow Java CI to pass

thomasjlalba committed:
Add Non-functional Requirements in Developer Guide

avielcx committed:
Merge pull request #245 from thomasjlalba/Thomas-JavaCI

Update runtest's expected output to allow Java CI to pass

avielcx committed:
Merge pull request #246 from thomasjlalba/Thomas-NFR

Add Non-functional Requirements in Developer Guide

hyperbola-bear committed:
test for currency stuff

hyperbola-bear committed:
Merge pull request #13 from AY2223S2-CS2113-T12-1/master

update

hyperbola-bear committed:
Merge pull request #247 from hyperbola-bear/master

currency test

Rayleigh47 committed:
edit PPP further

Rayleigh47 committed:
Merge branch 'master' into ppp-update

avielcx committed:
Merge pull request #249 from Rayleigh47/ppp-update

Update PPP

hyperbola-bear committed:
Merge pull request #14 from AY2223S2-CS2113-T12-1/master

update

avielcx committed:
Update UG

avielcx committed:
Update UG for "find"

Rayleigh47 committed:
added char limit to category and description fields

Rayleigh47 committed:
edit add use case in UG for new char limits

avielcx committed:
Make file read-only

Rayleigh47 committed:
added message for empty expense or income list

Rayleigh47 committed:
clean up ListExpenseCommand and ListIncomeCommand to use Ui

Rayleigh47 committed:
Merge pull request #250 from avielcx/master

Update UG

Rayleigh47 committed:
Merge branch 'master' into bugfix-PED

Rayleigh47 committed:
resolve merge conflict and gradle check

Rayleigh47 committed:
fixed incorrect exception message for category field character limit

avielcx committed:
Merge pull request #251 from Rayleigh47/bugfix-PED

Bugfix ped

avielcx committed:
Revert Storage class changes

avielcx committed:
Update PPP

hyperbola-bear committed:
tried unicode proof, + currency issues

avielcx committed:
Fix bug in PPP

Rayleigh47 committed:
Merge pull request #252 from avielcx/master

Revert Storage class changes, update PPP

hyperbola-bear committed:
Merge pull request #15 from AY2223S2-CS2113-T12-1/master

update

thomasjlalba committed:
Update error handling for add expense command to make error messages more specific

thomasjlalba committed:
Update error message expected for JUnit test

hyperbola-bear committed:
fixed my expense parser to stop unicode??

hyperbola-bear committed:
oops checkstyle

thomasjlalba committed:
Update error handling for add income command to make error messages more specific

thomasjlalba committed:
Update JUnit test with updated expected message
Reformat code to match indentation standards

thomasjlalba committed:
Change error messages to make it more specific
Remove redundant try-catches

hyperbola-bear committed:
plant uml for set currency and livecurrency api in DG

thomasjlalba committed:
Edit error message to make it more specific

thomasjlalba committed:
Add JUnit tests for Incomes and Expenses classes

thomasjlalba committed:
Add documentation for new JUnit classes

thomasjlalba committed:
Merge pull request #253 from hyperbola-bear/master

Currency issue and unicode maybe?

thomasjlalba committed:
Add other notes section in User Guide

avielcx committed:
Merge branch 'master' into Thomas-ErrorMessage

avielcx committed:
Merge pull request #254 from thomasjlalba/Thomas-ErrorMessage

Update error message to make it more specific

avielcx committed:
Revert "Update error message to make it more specific"

avielcx committed:
Merge pull request #255 from AY2223S2-CS2113-T12-1/revert-254-Thomas-ErrorMessage

Revert "Update error message to make it more specific"

avielcx committed:
Fix Incomes and Expenses class

avielcx committed:
Merge branch 'master' into master

avielcx committed:
Merge pull request #256 from avielcx/master

Fix Incomes and Expenses class

thomasjlalba committed:
Add instructions for manual testing section. Included startup, add, edit commands. Also added headers for the rest of the commands.

thomasjlalba committed:
Add delete/clear command to the instructions for manual testing

avielcx committed:
Fix merge conflicts

avielcx committed:
Merge pull request #257 from avielcx/master

Fix merge conflicts

thomasjlalba committed:
Add list and balance commands to manual testing of Developer Guide

avielcx committed:
Fix Checkstyle

avielcx committed:
Merge pull request #258 from avielcx/master

Fix Checkstyle

avielcx committed:
Fix checkstyle

avielcx committed:
Merge pull request #259 from avielcx/master

Fix checkstyle

avielcx committed:
Fix Junit bug

avielcx committed:
Merge pull request #260 from avielcx/master

Fix Junit bug

hyperbola-bear committed:
Merge pull request #16 from AY2223S2-CS2113-T12-1/master

update

Rayleigh47 committed:
round value for income and expense to 2dp before adding into respective array list

Rayleigh47 committed:
edit entry for add in user guide for value being stored as a 2dp double

Rayleigh47 committed:
Merge pull request #261 from thomasjlalba/Thomas-Docs

Update Manual testing instructions in Developer Guide

Rayleigh47 committed:
update exception message thrown for parsing value

hyperbola-bear committed:
lower bound checker for add income/expense

Rayleigh47 committed:
resolved merge conflict in UG

hyperbola-bear committed:
Merge pull request #17 from AY2223S2-CS2113-T12-1/master

update

Rayleigh47 committed:
edit UG

hyperbola-bear committed:
checkstyle

hyperbola-bear committed:
Merge branch 'master' of https://github.com/hyperbola-bear/tp

thomasjlalba committed:
Add JUnit tests for Expenses and Incomes classes from the parser package

avielcx committed:
Merge pull request #263 from hyperbola-bear/master

lower boundary check for add income/expense changed to 0.01

avielcx committed:
Merge pull request #262 from Rayleigh47/decimalfix-PED

Change entries for value to fixed 2dp

avielcx committed:
Update DG diagrams

avielcx committed:
Merge branch 'master' of https://github.com/AY2223S2-CS2113-T12-1/tp

hyperbola-bear committed:
Merge pull request #264 from avielcx/master

Update DG Diagrams

hyperbola-bear committed:
Merge pull request #18 from AY2223S2-CS2113-T12-1/master

update

thomasjlalba committed:
Add and edit error messages to make it more specific.
Add JUnit tests to correspond to the new error messages.
Changed limit of each expense/income to be less than 1,000,000

thomasjlalba committed:
Merge branch 'master' into Thomas-ErrorMessageAgain

Rayleigh47 committed:
Changed value field for adding income and expense to only accept values up to 2 d.p. or less

Rayleigh47 committed:
Update user guide to reflect accepting only 2 d.p. entries for value field

hyperbola-bear committed:
Merge pull request #265 from thomasjlalba/Thomas-ErrorMessageAgain

Add and update error messages to make it more specific

hyperbola-bear committed:
Merge pull request #19 from AY2223S2-CS2113-T12-1/master

update

Rayleigh47 committed:
cleaned up .puml file for AddIncome and ListExpense

Rayleigh47 committed:
resolved merge conflicts

hyperbola-bear committed:
Merge pull request #268 from Rayleigh47/decimalfix-PED

Decimal fixed to 2 d.p.

hyperbola-bear committed:
Merge pull request #20 from AY2223S2-CS2113-T12-1/master

update

thomasjlalba committed:
Fix bug where edit expense does not limit value to 999999.99

thomasjlalba committed:
Add JUnit tests for ExpenseList and IncomeList tests
Standardise the error messages in add command and edit command

hyperbola-bear committed:
Merge pull request #269 from thomasjlalba/Thomas-EditCommandBug

Fix bug where edit expense does not limit value to 999999.99

hyperbola-bear committed:
Merge pull request #21 from AY2223S2-CS2113-T12-1/master

update

hyperbola-bear committed:
upgrade find command

avielcx committed:
Merge pull request #270 from hyperbola-bear/master

upgraded find command

thomasjlalba committed:
Rename PPP file to match github username

Rayleigh47 committed:
Merge pull request #271 from thomasjlalba/master

Rename PPP file to match github username

Rayleigh47 committed:
Update Yan Ming's PPP to github name

hyperbola-bear committed:
update DG, UG and error checking for Find

hyperbola-bear committed:
checkstyle

hyperbola-bear committed:
Merge pull request #22 from AY2223S2-CS2113-T12-1/master

update

hyperbola-bear committed:
Merge pull request #272 from Rayleigh47/master

Update Yan Ming's PPP to github name

Rayleigh47 committed:
Merge pull request #273 from hyperbola-bear/master

updated DG, UG and some checks for Find

Rayleigh47 committed:
cleanup parser class and further javadoc

Rayleigh47 committed:
clean up PPP

Rayleigh47 committed:
update UI help command for add target command

Rayleigh47 committed:
handle values for setting of target to follow constraints

Rayleigh47 committed:
allow for negative values to be parsed by set target command

Rayleigh47 committed:
edit ym PPP

Rayleigh47 committed:
further edit to UG and DG to tally with set target restraints

Rayleigh47 committed:
sort out checkstyle

Rayleigh47 committed:
cleanup type for clear expense

avielcx committed:
Update UG, DG. Add ClearTargetCommand

avielcx committed:
Merge pull request #276 from avielcx/master

Update UG, DG. Add ClearTargetCommand

avielcx committed:
Update Target Limit

Rayleigh47 committed:
cleanup DG to add new commands to header table

avielcx committed:
Merge pull request #277 from avielcx/master

Update Target Limit

Rayleigh47 committed:
clean up uml diagrams for add income and list expense

Rayleigh47 committed:
resolved merge conflicts

Rayleigh47 committed:
further clean up of uml diagrams

Rayleigh47 committed:
Added architecture diagram and update design portion of DG

thomasjlalba committed:
update Developer Guide for version 2.1

thomasjlalba committed:
Update EditExpenseCommand and EditIncomeCommand classes to remove magic strings

thomasjlalba committed:
Merge pull request #275 from Rayleigh47/cleanup-v2.1

Cleanup UG, DG, PPP and set target command

thomasjlalba committed:
Merge branch 'master' into Thomas-DG

thomasjlalba committed:
Update parser to make the command and fields for arguments case insensitive

thomasjlalba committed:
Update User Guide to note on the case insensitivitiy

Rayleigh47 committed:
Merge pull request #278 from thomasjlalba/Thomas-DG

Update Developer Guide for Version 2.1

hyperbola-bear committed:
Merge pull request #279 from thomasjlalba/Thomas-CaseInsensitive

Update parser to allow command and argument field to be case insensitive

hyperbola-bear committed:
updated livecurrencyapi in DG

hyperbola-bear committed:
Merge branch 'master' into master

hyperbola-bear committed:
Merge pull request #23 from AY2223S2-CS2113-T12-1/master

update

avielcx committed:
Add Junit for ClearTargetCommand. Update UG, DG, PPP

avielcx committed:
Merge pull request #281 from avielcx/master

Add Junit for ClearTargetCommand. Update UG, DG, PPP

avielcx committed:
Merge pull request #280 from hyperbola-bear/master

update livecurrencyapi on DG

avielcx committed:
Improve language use in DG and UG

avielcx committed:
Merge pull request #282 from avielcx/master

Improve language use in DG and UG

avielcx committed:
Fix typo

avielcx committed:
Fix checkstyle

avielcx committed:
Merge pull request #283 from avielcx/master

fix typo

avielcx committed:
fix checkstyle

avielcx committed:
Merge pull request #284 from avielcx/master

fix checkstyle

avielcx committed:
Logging

avielcx committed:
fix checkstyle

avielcx committed:
logging

avielcx committed:
Update TOC

Rayleigh47 committed:
Merge pull request #285 from avielcx/master

Logging

Rayleigh47 committed:
Implement logger and log file for Storage

avielcx committed:
Merge pull request #286 from Rayleigh47/logger-test

Implement logger and log file for Storage

avielcx committed:
Logging

avielcx committed:
update logging

avielcx committed:
fix checkstyle

avielcx committed:
Fix checkstyle

Rayleigh47 committed:
Merge pull request #287 from avielcx/master

Logging

avielcx committed:
Update PPP, Product Website, AboutUs Image

avielcx committed:
Merge pull request #288 from avielcx/master

Update PPP, Product Website, AboutUs Image

thomasjlalba committed:
Fix errors with documentation in User Guide
Update thomasjlalba PPP

avielcx committed:
Update logging

avielcx committed:
Merge pull request #289 from avielcx/master

Update logging

avielcx committed:
Update Logging

avielcx committed:
Update Logging

avielcx committed:
Merge pull request #291 from avielcx/master

Update Logging

thomasjlalba committed:
Merge pull request #290 from thomasjlalba/master

Fix errors with documentation in User Guide

thomasjlalba committed:
makes the UG and DG clearer regarding extra field inputs

avielcx committed:
Fix TOC

avielcx committed:
Merge pull request #293 from avielcx/master

Fix TOC

avielcx committed:
Update TOC

avielcx committed:
Merge pull request #294 from avielcx/master

Update TOC

avielcx committed:
Update DG TOC

avielcx committed:
Merge pull request #295 from avielcx/master

Update DG TOC

avielcx committed:
Merge pull request #292 from thomasjlalba/Thomas-Doc

Update UG and DG clearer regarding extra field inputs

thomasjlalba committed:
Fix erratas in thomasjlalba PPP and Developer Guide

avielcx committed:
Merge pull request #296 from thomasjlalba/Thomas-Erratas

Fix errata in thomasjlalba PPP and Developer Guide

avielcx committed:
Update PPP

avielcx committed:
Merge pull request #297 from avielcx/master

Update PPP

avielcx committed:
Update PPP

avielcx committed:
Merge pull request #298 from avielcx/master

Update PPP

avielcx committed:
Update PPP

avielcx committed:
Merge pull request #299 from avielcx/master

Update PPP

avielcx committed:
Update PPP

avielcx committed:
Merge pull request #300 from avielcx/master

Update PPP

avielcx committed:
Update UG to inform users not to edit any txt files

thomasjlalba committed:
Fix errors made in User Guide

thomasjlalba committed:
Merge pull request #301 from thomasjlalba/Thomas-DocAgain

Fix errors made in User Guide

avielcx committed:
Update UG

avielcx committed:
Update DG

Rayleigh47 committed:
Update UserGuide.md

Rayleigh47 committed:
Merge pull request #302 from avielcx/master

Update DG, UG

hyperbola-bear committed:
Merge pull request #24 from AY2223S2-CS2113-T12-1/master

update

hyperbola-bear committed:
remove comments

thomasjlalba committed:
Clean up code to improve code quality

hyperbola-bear committed:
checkstyle

Rayleigh47 committed:
Cleaned up test files for code quality

avielcx committed:
Improve code quality

avielcx committed:
Merge pull request #306 from avielcx/master

Improve code quality

Rayleigh47 committed:
Merge pull request #305 from Rayleigh47/master

Cleaned up test files for code quality

Rayleigh47 committed:
Merge pull request #304 from hyperbola-bear/master

clean up non-javadoc comments

thomasjlalba committed:
Fix checkstyle error

thomasjlalba committed:
Merge pull request #303 from thomasjlalba/Thomas-CleanCode

Clean up code to improve code quality

hyperbola-bear committed:
Merge pull request #25 from AY2223S2-CS2113-T12-1/master

final testing

Rayleigh47 committed:
update ym PPP

Rayleigh47 committed:
Merge branch 'master' of https://github.com/AY2223S2-CS2113-T12-1/tp

avielcx committed:
Update TOC

Rayleigh47 committed:
Merge pull request #307 from avielcx/master

Update TOC

thomasjlalba committed:
Update instructions/command line to be trimmed to cut off excess spaces

Rayleigh47 committed:
Merge branch 'master' of https://github.com/AY2223S2-CS2113-T12-1/tp

Rayleigh47 committed:
Merge pull request #308 from thomasjlalba/Thomas-TrimCommand

Update instructions/command line to be trimmed to cut off excess spaces

Rayleigh47 committed:
Merge branch 'master' of https://github.com/AY2223S2-CS2113-T12-1/tp

hyperbola-bear committed:
added currency to target

hyperbola-bear committed:
Merge pull request #26 from AY2223S2-CS2113-T12-1/master

update

Rayleigh47 committed:
Remove consolehandler logging

avielcx committed:
Merge pull request #309 from hyperbola-bear/master

currency for target

Rayleigh47 committed:
Merge branch 'master' of https://github.com/AY2223S2-CS2113-T12-1/tp

Rayleigh47 committed:
Merge branch 'master' into final-update

thomasjlalba committed:
Merge pull request #310 from Rayleigh47/final-update

Remove console handler for logging

hyperbola-bear committed:
ppp

hyperbola-bear committed:
Merge pull request #27 from AY2223S2-CS2113-T12-1/master

update

hyperbola-bear committed:
Merge branch 'master' of https://github.com/hyperbola-bear/tp

Rayleigh47 committed:
Merge pull request #311 from hyperbola-bear/master

v2.1 wrap up

Rayleigh47 committed:
update show target to be 2dp

Rayleigh47 committed:
update DG for limit on number of records for expenses and incomes

avielcx committed:
Update PPP

avielcx committed:
Merge pull request #312 from avielcx/master

Update PPP

Rayleigh47 committed:
fixed show target test bug

Rayleigh47 committed:
Merge branch 'master' into final-update

thomasjlalba committed:
Add page break for PPP

thomasjlalba committed:
Add page breaks for Developer Guide

thomasjlalba committed:
Merge pull request #313 from Rayleigh47/final-update

Update show target to 2dp

avielcx committed:
Update UG

avielcx committed:
Merge pull request #314 from thomasjlalba/Thomas-PageBreaks

Thomas page breaks

Rayleigh47 committed:
Merge pull request #315 from avielcx/master

Update UG

thomasjlalba committed:
Add page break to neaten make Developer Guide pdf compatible

thomasjlalba committed:
Merge pull request #316 from thomasjlalba/Thomas-PageBreakAgain

Add page break to neaten make Developer Guide pdf compatible

Rayleigh47 committed:
Update rayleigh47.md

hyperbola-bear committed:
page break

hyperbola-bear committed:
Merge pull request #28 from AY2223S2-CS2113-T12-1/master

update

hyperbola-bear committed:
Merge pull request #317 from hyperbola-bear/master

Ui.png

Rayleigh47 committed:
Update UserGuide.md

Rayleigh47 committed:
Update DeveloperGuide.md

hyperbola-bear committed:
Merge pull request #29 from AY2223S2-CS2113-T12-1/master

update

hyperbola-bear committed:
category find

avielcx committed:
Update PPP

thomasjlalba committed:
Merge pull request #318 from hyperbola-bear/master

category find

avielcx committed:
Merge pull request #319 from avielcx/master

Update PPP
